ci/get-engine.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+if [[ -n "${DEBUG-}" ]]; then
+    set -x
+fi
+
+pushd "$(dirname "$0")"
+    # Unreal path is the path to the Engine directory. No symlinking for mac, because they seem to cause issues during the build. This should ultimately resolve to "/Users/buildkite-agent/builds/<agent name>/improbable//UnrealEngine".
```

Comment:
```suggestion
    # Unreal path is the path to the Engine directory. No symlinking for mac, because they seem to cause issues during the build. This should ultimately resolve to "/Users/buildkite-agent/builds/<agent name>/improbable/UnrealEngine".
```